android/src/app/organicmaps/routing/RoutingController.java

```diff
@@ -839,6 +844,34 @@ public boolean setEndPoint(@Nullable MapObject point)
     return true;
   }
 
+  public boolean continueToPoint(@NonNull MapObject point)
+  {
+    MapObject startPoint = getStartPoint();
+    MapObject endPoint = getEndPoint();
+
+    final Pair<String, String> description = getDescriptionForPoint(point);
+    Framework.nativeContinueRouteToPoint(description.first /* title */, description.second /* subtitle */,
+                                         0 /* intermediateIndex */, MapObject.isOfType(MapObject.MY_POSITION, point),
+                                         point.getLat(), point.getLon());
+
+    if (point.sameAs(startPoint))
+    {
+      if (endPoint == null)
+      {
+        Logger.d(TAG, "setEndPoint: skip because end point is empty");
+        return false;
+      }
+
+      Logger.d(TAG, "setEndPoint: swap with starting point");
```

Comment:
nit: I think these debug logs are useless, no?

Reply:
Removed debug logs in `RoutingController.setEndPoint()` and `RoutingController.continueToPoint()` methods.